static/app/debug/notifications/types.ts

```diff
@@ -20,6 +20,14 @@ export interface NotificationTemplateRegistration {
       subject: string;
       text_content: string;
     };
+    [NotificationProviderKey.SLACK]: {
+      blocks: Array<{
+        [key: string]: any; // Allow any additional unknown properties
+        type: string;
+        block_id?: string;
+        text?: {text: string; type: 'plain_text' | 'mrkdwn'};
+      }>;
```

Comment:
We don't actually use any of these properties since we just paste the whole blob into the URL link, so you could simplify this type to one-line: `blocks: Record<string, any>[]`